src/trades/dto/create-trade.input.ts

```diff
@@ -0,0 +1,25 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class CreateTradeInput {
+  @Field()
+  trade_uuid: string;
```

Comment:
Let's generate the uuid in the creation function itself, like the way I'm doing it for items in my PR.